src/Router/RabbitMq/Adapter.php

```diff
@@ -147,4 +167,36 @@ private function handleException(Message $message, \Throwable $e): void
             $message->nack()->subscribe(null, null, $onCompleted);
         }
     }
+
+    private function handleUnprocessableException(Message $message, UnprocessableException $e): void
```

Comment:
handleRejectException ?

Reply:
The exception is called UnprocessableException, therefore i'd better let this function's name as it is.

Reply:
Ok but I thought the name is missunderstanding at the reading because the consequence is a reject, and i must read twice to understand the necessity of this exception.
At the beginning i thought it was the general exception for nacking.

Reply:
Alright, should i call the exception "RejectException" then ?

Reply:
Yes it's clearly more adapted